lib/node.js

```diff
@@ -524,16 +524,26 @@ declare module "readline" {
 }
 
 declare class stream$Stream extends events$EventEmitter {
-  pipe(dest: any, options: any): void;
+  pipe(dest: any, options?: any): void;
 }
 
 declare class stream$Readable extends stream$Stream {
   setEncoding(): void;
+  isPaused(): boolean;
+  pause(): void;
+  read(size?: number): ?(string | Buffer);
+  resume(): void;
+  unpipe(dest: any): void;
```

Comment:
Looks like the argument here is optional, right? https://nodejs.org/api/stream.html#stream_readable_unpipe_destination